src/NServiceBus.Core/Routing/Routers/UnicastSendRouterConnector.cs

```diff
@@ -22,7 +22,7 @@ public enum RouteOption
         public UnicastSendRouterConnector(
             string sharedQueue,
             string instanceSpecificQueue,
-            IUnicastRouter unicastRouter,
+            IUnicastSendRouter unicastRouter,
```

Comment:
I'd rename this and the field to `unicastSendRouter` to match the change to the `UnicastPublishRouterConnector `.